metricbeat/module/syncgateway/db/data_test.go

```diff
@@ -0,0 +1,38 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package db
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/elastic/beats/v7/metricbeat/module/syncgateway"
+
+	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
+)
+
+func TestData(t *testing.T) {
```

Comment:
Is there any way to add `TestMetricsetFieldsDocumented` somewhere for this module? For example: https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/azure/compute_vm/compute_vm_integration_test.go#L31

Reply:
Cool! I added them all but in `db` metricset as it seems there's some bug in the testing script where it doesn't recognize some fields as being documented even when they are there (I checked like.... 10 times 😅  )

I'll take a look at the script once I finish with few things.